client/src/pages/pot-instance/PotInstance.tsx

```diff
@@ -8,17 +8,17 @@ import { PotContent } from 'pages/pot-instance/models';
 import styles from './styles.module.css';
 
 const PotInstance: FC = () => {
-  const [potType, usePotType] = useState(0);
   const [hotpotStart, setHotpotStart] = useState(false);
   const [cookedPotContent, setCookedPotContent] = useState<PotContent[]>([]);
+  const [soupbase, useSoupbase] = useState(['']);
 
-  // functions to update the landing page
-  const updatePotType = (type: number) => {
-    usePotType(type);
-  };
-
-  const startHotPot = () => {
+  const startHotPot = (newSoupbase: string[]) => {
     setHotpotStart(true);
+    const tempSoupbase = newSoupbase;
+    if (tempSoupbase.length < 2) {
```

Comment:
Is there a way we can avoid this magic number, like referencing a list length somewhere?

Reply:
Updated with a constant for potImages along with comments.